tests/test_fixtures/rule_based_profiler/example_notebooks/SelfInitializingExpectations.ipynb

```diff
@@ -0,0 +1,886 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "cda410dd-b44e-4456-b04b-57537b635654",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ruamel import yaml\n",
+    "import great_expectations as ge\n",
+    "from great_expectations.core.batch import BatchRequest\n",
+    "from great_expectations.expectations.expectation import Expectation\n",
+    "from great_expectations.rule_based_profiler.config import RuleBasedProfilerConfig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a25cb0ae-94d9-4584-ac3a-6d5685ca2201",
+   "metadata": {},
+   "source": [
+    "# Self-Initializing Expectations\n",
+    "- Self-initializing `Expectations` utilize `RuleBasedProfilers` to automate parameter estimation for Expectations using a Batch or Batches that have been loaded into a `Validator`. \n",
+    "\n",
+    "### Do they work for all `Expectations`?\n",
+    "- No, not all `Expectations` have parameters that can be estimated. As an example, `ExpectColumnToExist` only takes in a `Domain` (which is the column name)  and outputs whether the values in the column are `None`. It would be an example of an `Expectation` that would not work under the self-initializing framework. \n",
```

Comment:
I think ExpectColumnToExist just checks whether the column name is in the list of names in column metadata, it doesn't check the values in the column.

Reply:
Seconding this.  It seems non intuitive for it to also check the contents of the column -- I would expect that to be ExpectColumnToBePopulated or something.

Reply:
Thank you @anthonyburdi. I'll update this description.

Reply:
Thanks for catching this @anthonyburdi and @Rachel-Reverie.. I had originally used both `ExpectColumnToExist` and `ExpectColumnValuesToNotBeNull`, and apparently frankensteined both descriptions together 😓